package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@axelar-network/axelarjs-sdk",
   "version": "0.4.24-alpha.1",
   "description": "The JavaScript SDK for Axelar Network",
-  "main": "dist/src/index.js",
```

Comment:
i noticed we collapsed the nested dist builds so amended the package.json file accordingly